vm/chef/cookbooks/redash/files/redash-setup

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash -eu
+#
+# Copyright 2021 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+source /opt/c2d/c2d-utils || exit 1
+
+REDASH_BASE_PATH="/opt/redash"
+if [[ -e $REDASH_BASE_PATH/env ]]; then
+    rm $REDASH_BASE_PATH/env
+    touch $REDASH_BASE_PATH/env
+fi
+
+readonly redash_admin_name="$(get_attribute_value "redash_admin_name")"
+readonly redash_admin_password="$(get_attribute_value "redash_admin_password")"
+readonly redash_admin_email="$(get_attribute_value "redash_admin_email")"
+
+readonly redash_cookie_key="$(get_attribute_value "redash_cookie_key")"
+readonly redash_secret_key="$(get_attribute_value "redash_secret_key")"
+readonly redash_db_password="$(get_attribute_value "redash_db_password")"
+readonly redash_database_url="postgresql://postgres:${redash_db_password}@postgres/postgres"
+
+echo "PYTHONUNBUFFERED=0" >> $REDASH_BASE_PATH/env
+echo "REDASH_LOG_LEVEL=INFO" >> $REDASH_BASE_PATH/env
+echo "REDASH_REDIS_URL=redis://redis:6379/0" >> $REDASH_BASE_PATH/env
+echo "POSTGRES_PASSWORD=${redash_db_password}" >> $REDASH_BASE_PATH/env
+echo "REDASH_COOKIE_SECRET=${redash_cookie_key}" >> $REDASH_BASE_PATH/env
+echo "REDASH_SECRET_KEY=${redash_secret_key}" >> $REDASH_BASE_PATH/env
+echo "REDASH_DATABASE_URL=${redash_database_url}" >> $REDASH_BASE_PATH/env
+
+echo "export COMPOSE_PROJECT_NAME=redash" >> ~/.profile
+echo "export COMPOSE_FILE=/opt/redash/setup/docker-compose.yml" >> ~/.profile
```

Comment:
I'm not sure if you can modify /root home folder. If check fails, probably you'll have to move this to startup script.

Reply:
I changed the location of those variables, I added them to /etc/profile